### DISCUSSION TITLE
How can I get inline on the code for version 9.0?

### DISCUSSION THREAD

**Discussion by g0su911** [remarkjs/remark#1243] [Q&A] [answered]
![image](https://github.com/remarkjs/remark/assets/12660636/8fe27ab4-9d53-4cee-8257-74f1a927434e)
backquote is not removed. how can I removed it?

This is my component
```js
        code(props) {
          const { node, children, className, ref, ...rest } = props;
          const match = /language-(\w+)/.exec(className || "");
          return match ? (
            <SyntaxHighlighter
              children={String(children).replace(/\n$/, "")}
              language={match[1]}
              {...rest}
              style={materialDark}
            />
          ) : (
            <code
              className={className ? className : "dark:text-white bg-gray-200"}
              {...rest}
            >
              {children}
            </code>
          );
        },
```

And I read a patch note but I don't know how to use a inline code property.
What is mean `inline on code — create a plugin or use pre for the block`. I can't understand it. Just code a pre component?

**Comment by wooorm** [comment, marked as answer]
You use tailwind, *thats* what shows the `` ` ``; fix your CSS setup

**Reply by g0su911** [reply]
good. Thank you. I fixed my tailwind css config.
